docs/Contributing/Release.md

```diff
@@ -0,0 +1,241 @@
+# Release Process
+
+## Versioning
+- Major stable public release versions require a major number increase
+- Breaking changes and major features require a minor number increase
+- Bugfixes require a patch number increase
+
+## Remotes
+> **NOTE:** Regarding git remotes:
+> * `upstream` points to core repository location
+> * `origin` points to the user's fork
```

Comment:
Ah, well, I personally have `origin` as the original and my own or other forks as `$USERNAME` (`haraldh`).

Reply:
<aybe I should set this behind a variable then. It's more of an assumption for the sake of documentation purposes.